scripts/suite.py

```diff
@@ -126,6 +127,9 @@
                               2/<outof> ... <outof>-1/<outof> will schedule all
                               jobs in the suite (many more than once). If specified,
                               this value can be found in results.log.
+  --descr <descr>             Use to rerun tests. Reschedule test based on their
```

Comment:
@zmc what was the outcome after talk to author? Was our suggestion correct the feature more about scheduling a new suite rather than rerun?

Reply:
In that case it must look like:
--descr <descr> Schedule a suite based on comma separated list of descriptions.

Reply:
@kshtsk This is why english-ish users should get their texts double checked.